src/main/java/example/security/ElideStandaloneExampleSecurityContext.java

```diff
@@ -0,0 +1,37 @@
+package example.security;
```

Comment:
This and the next file are mossing copyright.  Instead of fixing that though, I'm wondering if we could make these anonymous classes and somehow roll them into the Auth filter.  They seem bound to it and are not generally useful outside of it.